contrib/corrosion-cmake/config.toml.in

```diff
@@ -1,6 +1,9 @@
 [env]
+# Used by skim
 CC = "@CMAKE_C_COMPILER@"
+CC_@Rust_CARGO_TARGET@ = "@CMAKE_C_COMPILER@"
```

Comment:
Interesting, `Rust_CARGO_TARGET` contains `-`, but it looks like it is automatically replaced with `_`.

Also I see the following in corrosion:

```
if(CMAKE_C_COMPILER)
    # This variable is read by cc-rs (often used in build scripts) to determine the c-compiler.
    # It can still be overridden if the user sets the non underscore variant via the environment variables
    # on the target.
    list(APPEND corrosion_cc_rs_flags "CC_${_CORROSION_RUST_CARGO_TARGET_UNDERSCORE}=${CMAKE_C_COMPILER}")
endif()
if(CMAKE_CXX_COMPILER)
    list(APPEND corrosion_cc_rs_flags "CXX_${_CORROSION_RUST_CARGO_TARGET_UNDERSCORE}=${CMAKE_CXX_COMPILER}")
endif()
```

So I guess, we don't need it

Reply:
I'm not sure if this works in all cases. When I set up `CXX = "nonExistingClang++"` it would not fail to build skim. Needs more investigation but I'll leave it to revisit it once we remove things from `config.toml.in` if possible.

Reply:
Did not get this. As you commented `CXX` should not work, only `CXX_@Rust_CARGO_TARGET@ `

Reply:
Yes. I thought you meant the opposite. CXX appeared to do nothing in our current build, but it's extremely hard to know if some other build tool uses it. For example, some crates could use the CXX via `build.rs` (all I've seen are full of hacks)

Reply:
Just to ensure that we are on the same page - I'm suggesting removing `CC_@Rust_CARGO_TARGET@` and `CXX_@Rust_CARGO_TARGET@` from our `config.toml.in` since corrosion has it already

> For example, some crates could use the CXX via build.rs

Hm, I guess it is possible (if you will not forget, please add a comment about this in the next PR)